src/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.html

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="slds-grid">
+    <div class="slds-var-p-top_x-small">
+      <template if:true={isFull}>
+        <lightning-progress-ring
+          size="large"
+          value={progress}
+          variant="base-autocomplete"
+        ></lightning-progress-ring>
+      </template>
+      <template if:false={isFull}>
+        <div
+          class="slds-progress-ring slds-progress-ring_large slds-progress-ring_warning"
+        >
+          <div
+            class="slds-progress-ring__progress"
+            role="progressbar"
+            aria-valuemin="0"
+            aria-valuemax="100"
+            aria-valuenow={progress}
+          >
+            <svg viewBox="-1 -1 2 2">
+              <path class="slds-progress-ring__path" d={progressPath}></path>
+            </svg>
+          </div>
```

Comment:
Just because the lwc component when used in the `warning variant` which shows the progress in orange, always displays a warning icon inside the progress ring and thats not what we want following our design. So I used the SLDS progress ring  blueprint to avoid having this unwanted  icon and to respect the design.